docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,303 @@
+* The Go import path of the CR Type definition
+* The Custom Resource Type you want to depend on.
+
+The Domain and Group variables have been discussed in other parts of the documentation.  The import path would be located in the project that installs the CR.
+The Custom Resource Type is usually a Go Type of the same name as the CustomResourceDefinition in kubernetes, e.g. for a `Pod` there will be a type `Pod` in the `v1` group.
+For Kubernetes Core Types, the domain can be omitted.
+``
+This document uses `my` and `their` prefixes as a naming convention for repos, groups, and types to clearly distinguish between your own project and the external one you are referencing.
+
+In our example we will assume the following external API Type:
+
+`github.com/theiruser/theirproject` is another kubebuilder project on whose CRD we want to depend and extend on.
+Thus, it contains a `go.mod` in its repository root. The import path for the go types would be `github.com/theiruser/theirproject/api/theirgroup/v1alpha1`.
+
+The Domain of the CR is `theirs.com`, the Group is `theirgroup` and the kind and go type would be `ExternalType`.
+
+If there is an interest to have multiple Controllers running in different Groups (e.g. because one is an owned CRD and one is an external Type), please first
+reconfigure the Project to use a multi-group layout as described in the [Multi-Group documentation](../migration/multi-group.md).
+
+### Prerequisites
+
+The following guide assumes that you have already created a project using `kubebuilder init` in a directory in the GOPATH. Please reference the [Getting Started Guide](../getting-started.md) for more information.
+
+Note that if you did not pass `--domain` to `kubebuilder init` you will need to modify it for the individual api types as the default is `my.domain`, not `theirs.com`.
+Similarly, if you intend to use your own domain, please configure your own domain with `kubebuilder init` and do not use `theirs.com for the domain.
+
+### Add a controller for the external Type
+
+#### Using a type not included in the Kubernetes Core Resources
+
+We will now add a controller for the given type
+
+```shell
+kubebuilder create api --group theirgroup --version v1alpha1 --kind ExternalType --controller --resource=false
+```
+
+Note that the `resource` argument is set to false, as we are not attempting to create our own CustomResourceDefinition,
+but instead rely on an external one.
+
+This will result in a `PROJECT` entry with the default domain of the `PROJECT` (`my.domain` if not specified in `kubebuilder init`).
+For use of other domains, such as `theirs.com`, one will have to manually adjust the `PROJECT` file with the correct domain for the entry:
+
+file: PROJECT
+```
+domain: my.domain
+layout:
+- go.kubebuilder.io/v4
+projectName: testkube
+repo: github.com/jakobmoellerdev/testkube
+resources:
+- controller: true
+  domain: theirs.com
+  group: mygroup
+  kind: ExternalType
+  version: v1alpha1
```

Comment:
You need add the path

like we do for the core types

```
- controller: true
  group: apps
  kind: Deployment
  path: k8s.io/api/apps/v1
  version: v1
```

We no longer use domain. 
See that the old doc is very outdated.

Reply:
As i said before, path is not set by default at all when using the current main. If i try a kubebuilder init and a create api, it will not use path but domain.

Reply:
Hi @jakobmoellerdev 

Following the structure, see: (https://github.com/kubernetes-sigs/kubebuilder/blob/3c9cf656ce4d62a10c8e78a1c0f066de47c983db/testdata/project-v4/PROJECT#L10-L23)

```yaml
resources:
- api:
    crdVersion: v1
    namespaced: true
  controller: true
  domain: testproject.org <- Replace for the domain (valid only for external types, Core types will have this info by default when required)
  group: crew
  kind: Captain
  path: sigs.k8s.io/kubebuilder/testdata/project-v4/api/v1 <- Replace for the path/import (valid only for external types. Core types will have this info by default)
  version: v1
  webhooks:
    defaulting: true
    validation: true
    webhookVersion: v1
```